common/include/pcl/PCLImage.h

```diff
@@ -5,19 +5,20 @@
 #include <ostream>  // for ostream
 
 #include <pcl/PCLHeader.h>   // for PCLHeader
+#include <pcl/types.h> //for index_t
 
 namespace pcl
 {
   struct PCLImage
   {
      ::pcl::PCLHeader  header;
```

Comment:
Should we reorder the memory layout to make it more cache friendly? We're dumping away at API and ABI stability anyways

Reply:
Sure. What your proposal?

Reply:
Maybe no change. As I started describing, I saw it'd cost too much (in time) and not provide comparable benefits